### PR TITLE
Mark Patients as Deleted

### PR DESCRIPTION
**Pull request by AJSclyn**
Added new column for administrators to soft delete patients.  Patients that have been deleted will not be able to be searched in the Home Page, looked up in the Triage section, and the patient will not show up in the research data.

### CODE REVIEW

**Comment by kdunlap** [comment]
This code looks good. Just going to test to be sure.

**Comment by kevinzurek** [comment]
This is great on the back-end, but how do i delete a patient as an Administrator?!

https://teamfemr.atlassian.net/browse/FEMR-36

I think we could accept this as long as we break FEMR-36 down into sub-tasks to indicate that the UI doesn't yet support this.

**Comment by AJSclyn** [comment]
I can add a button when a patient looks at past encounter history that is only visible if the current user is an admin.  I was thinking right next to "View this Patient in Triage button".  Let me know before I implement the change.

**Comment by kevinzurek** [comment]
I can dig it

**Comment by AJSclyn** [comment]
image

File: conf/evolutions/default/86.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# --- !Ups
+
+ALTER TABLE `patients`
+ADD COLUMN `isDeleted` DATETIME NULL DEFAULT NULL AFTER `photo_id`;
+
+# --- !Downs
+
+ALTER TABLE `patients`;
+DROP COLUMN `isDeleted;
```
Comment (kevinzurek) [inline conf/evolutions/default/86.sql]: Just tried to execute the down script - 'isDeleted' needs a closing grave accent otherwise it won't work.

**Comment by kevinzurek** [comment]
I'm a bit concerned that this doesn't allow someone who accidentally clicks delete to undo their action. A user can only click "Ok" and not "cancel".

**Comment by AJSclyn** [comment]
image

**Comment by kevinzurek** [comment]
I merged this manually

**Comment by AJSclyn** [comment]
Yahoo ! :)

On Sat, Dec 5, 2015 at 8:01 PM, Kevin Zurek <notifications@github.com>
wrote:

> Closed #463 <https://github.com/FEMR/femr/pull/463>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FEMR/femr/pull/463#event-483582511>.
>



-- 
AJ Saclayan

**Pull request closed without merging**